Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -284,16 +287,16 @@ class AccountStatusSetting: WithAccountSetting {
 
         if let account = profile.getAccount() {
             switch account.actionNeeded {
-            case .needsVerification:
+            case .none, .needsVerification:
                 var cs = URLComponents(url: account.configuration.settingsURL, resolvingAgainstBaseURL: false)
                 cs?.queryItems?.append(URLQueryItem(name: "email", value: account.email))
                 viewController.url = try! cs?.asURL()
```

Comment:
This `try!` cannot be here. It will crash the app if an exception happens. Same below.